### DISCUSSION TITLE
Logs are not cleaned

### DISCUSSION THREAD

**Discussion by satiukov-e** [apache/airflow#31639] [Q&A] [answered]
Hello! I have set up Airflow in Docker using the airflow:2.6.1 image and added `AIRFLOW__LOG_RETENTION_DAYS: 1` to the `docker-compose.yaml` file to test log cleaning. However, the logs are not being cleaned. From my understanding, the `clean-logs.sh` script should be executed, but it doesn't appear to be running. Should it work?

**Comment by potiuk** [comment, marked as answer]
No. Clean-logs is script that you can run if you want - feel free to extend the compose for your own. clean-logs is  used in Helm Chart in log groomer container, but docker-compose is exclusively for quick start and you on your own should be responsible to modify it in the way that fits your case, because docker-compose of ours is quick-start and it is intended for people like you, who undoubtedly know what they are doing to use docker-compose to modify it as it fits them.
